test/test_heuristics.rb

```diff
@@ -84,14 +92,15 @@ def test_bb_by_heuristics
 
   def test_builds_by_heuristics
     assert_heuristics({
-      "Text" => all_fixtures("Text", "*.builds"),
+      "Text" => all_fixtures("Text"),
       "XML" => all_fixtures("XML", "*.builds")
-    })
+    }, alt_name="test.builds")
   end
 
   def test_ch_by_heuristics
     assert_heuristics({
-      "xBase" => all_fixtures("xBase", ".ch")
+      nil => all_fixtures("Charity", "*.ch"),
```

Comment:
Could you add a comment here to explain the `nil` (lack of heuristic rule for Charity)?